tests/integration/udf/test_count_dict_entropy.py

```diff
@@ -0,0 +1,29 @@
+"""
+Tests for count dict entropy UDF
+"""
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize("source_type", ["snowflake, spark"], indirect=True)
```

Comment:
Would it make sense to include databricks source type in these tests since that's where the UDFs will be changed later?

Reply:
i'll add them in all at once later once the `databricks_unity` source type is set up. i also presume we'll likely keep the non-unity databricks setup in case anyone potential users are still on that.